cmd/server.go

```diff
@@ -99,6 +100,7 @@ const (
 	DefaultBitbucketBaseURL = bitbucketcloud.BaseURL
 	DefaultDataDir          = "~/.atlantis"
 	DefaultGHHostname       = "github.com"
+	DefaultGHTeamAllowlist  = "*:*"
```

Comment:
I would rather this default null and force intentional configuration so that its safer to future add support for commands such as `import` and `rm` which these are no longer sane/safe assumptions. I'd personally prefer a breaking change over backwards compatibility here.